.github/workflows/main.yml

```diff
@@ -55,12 +55,13 @@ jobs:
           - python: "3.7"
             image_tag: "3.20"
           - python: "3.8"
+            pulp_cli: "0.16.0"
             image_tag: "3.19"
           - python: "3.9"
             image_tag: "3.18"
           - python: "3.10"
             pulp_cli: "0.15.0"
-            image_tag: "3.17"
+            image_tag: "3.22"
```

Comment:
Ok, I am tired of waiting on this without having a working state in `main` branch! pulp-cli itself does not yet test against 3.22, we do not need to be ahead of pulp-cli!

Let's separate this out into a separate PR, and get everything else merged today!

Reply:
Yep sorry for the delay. I agree waiting is not a good option at this time.